src/App.tsx

```diff
@@ -1,14 +1,59 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
+const defaultTodo = {
+  id: 0,
+  title: 'no title',
+  completed: false,
+  userId: 0,
+};
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoader, setIsLoader] = useState(true);
+  const [selectedTodo, setSelectedTodo] = useState<Todo>(defaultTodo);
+  const [selectValue, setSelectValue] = useState('');
+  const [filterValue, setFilterValue] = useState('');
+
```

Comment:
filterValue should be 'all' by default

Reply:
Do you mean selectValue?

Reply:
> Do you mean selectValue?

Yes, sorry....my mistake))))

Reply:
Yes, you're right. But in my solution I use `""` to display all todos)